tests/php/bug49102-1.phpt

```diff
@@ -42,73 +46,40 @@ Content:
 
-        <div class="methodsynopsis dc-description">
-            <span class="type">void</span> <span class="methodname"><strong>setIteratorMode</strong></span>
-             ( <span class="methodparam"><span class="type">int</span> <code class="parameter">$mode</code></span>
-         )</div>
+        <div class="methodsynopsis dc-description"><span class="methodname"><strong>setIteratorMode</strong></span>(<span class="methodparam"><span class="type">int</span> <code class="parameter">$mode</code></span>): <span class="type"><span class="type void">void</span></span></div>
 
 
         <div class="classsynopsisinfo classsynopsisinfo_comment">/* Inherited methods */</div>
-        <div class="methodsynopsis dc-description">
-            <span class="type">mixed</span> <span class="methodname"><strong>SplDoublyLinkedList::bottom</strong></span>
-             ( <span class="methodparam">void</span>
-         )</div>
-
-        <div class="methodsynopsis dc-description">
-            <span class="type">int</span> <span class="methodname"><strong>SplDoublyLinkedList::count</strong></span>
-             ( <span class="methodparam">void</span>
-         )</div>
-
-        <div class="methodsynopsis dc-description">
-            <span class="type">mixed</span> <span class="methodname"><strong>SplDoublyLinkedList::current</strong></span>
-             ( <span class="methodparam">void</span>
-         )</div>
-
-        <div class="methodsynopsis dc-description">
-            <span class="type">int</span> <span class="methodname"><strong>SplDoublyLinkedList::getIteratorMode</strong></span>
-             ( <span class="methodparam">void</span>
-         )</div>
-
-        <div class="methodsynopsis dc-description">
-            <span class="type">bool</span> <span class="methodname"><strong>SplDoublyLinkedList::offsetExists</strong></span>
-             ( <span class="methodparam"><span class="type"><a href="language.pseudo-types.html#language.types.mixed" class="type mixed">mixed</a></span> <code class="parameter">$index</code></span>
-         )</div>
-
-        <div class="methodsynopsis dc-description">
-            <span class="type">mixed</span> <span class="methodname"><strong>SplDoublyLinkedList::offsetGet</strong></span>
-             ( <span class="methodparam"><span class="type"><a href="language.pseudo-types.html#language.types.mixed" class="type mixed">mixed</a></span> <code class="parameter">$index</code></span>
-         )</div>
+        <div class="methodsynopsis dc-description"><span class="methodname"><strong>SplDoublyLinkedList::bottom</strong></span>(): <span class="type"><a href="language.types.declarations.html#language.types.declarations.mixed" class="type mixed">mixed</a></span></div>
+
+        <div class="methodsynopsis dc-description"><span class="methodname"><strong>SplDoublyLinkedList::count</strong></span>(): <span class="type">int</span></div>
```

Comment:
Showcasing how are links for primitive types are broken :D

Reply:
Oops, they sure are. :-) I'll take look in a follow up PR.

Reply:
I had https://github.com/php/phd/pull/33 which I started, *checks notes*, years ago :|